apps/wallet-mobile/src/features/WalletManager/wallet-manager.ts

```diff
@@ -383,6 +385,38 @@ export class WalletManager {
     await this.#rootStorage.setItem('deletedWalletIds', [])
   }
 
+  getWalletPlate(publicKeyHex: string): WalletChecksum {
+    const plate = walletChecksum(publicKeyHex)
+    return plate
+  }
+
+  getIsWalletDuplicated(publicKeyHex: string): boolean {
```

Comment:
```suggestion
  isWalletDuplicatedByXPub(publicKeyHex: string): boolean {
```

Reply:
is it the XPub, or accountKey?

Reply:
I think is the first internal addr, in this case `byAccPubHex`

Reply:
cc: @banklesss

Reply:
`isAccountDuplicated`

Reply:
done